attributes/packer_pipeline.rb

```diff
@@ -0,0 +1,6 @@
+default['osl-jenkins']['packer_pipeline']['openstack_taster_version'] = '0.0.2'
+default['packer']['version'] = '1.0.2'
```

Comment:
The general rule of thumb: If it's an upstream cookbook attribute, that should be in the recipe. If it's an attribute in the same namespace as the cookbook you're working on, that can be in the attributes.